### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,63 @@ class Cargo:
...
+    def go_down(self, step: int = 1) -> None:
+        self.coords[-1] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 max_load_weight: int,
+                 current_load: int
```
Comment (IvanStored) [inline app/main.py:53]: why DeliveryDrone doesn't accept `coords` argument?

**Review by IvanStored — CHANGES_REQUESTED** [review]

**Review by IvanStored — APPROVED** [review]

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,61 @@ class Cargo:
     def __init__(self, weight):
```
Comment (Nattalli) [inline app/main.py:2]: add type anotation

File: app/main.py [hunk 3]
```diff
@@ -2,4 +2,61 @@ class Cargo:
...
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None):
```
Comment (Nattalli) [inline app/main.py:10]: it's better to use `list[int]` here

File: app/main.py [hunk 4]
```diff
@@ -2,4 +2,61 @@ class Cargo:
...
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None):
```
Comment (Nattalli) [inline app/main.py:35]: it's better to use `list[int]` here

File: app/main.py [hunk 5]
```diff
@@ -2,4 +2,61 @@ class Cargo:
...
+        self.coords[-1] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 max_load_weight: int,
+                 current_load: int,
+                 coords: list = None):
```
Comment (Nattalli) [inline app/main.py:52]: it's better to use `list[int]` here

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Nattalli — APPROVED** [review]
Good job!

**Pull request closed without merging**